### PR TITLE
update client and magic based on new API design

### PR DESCRIPTION
**Pull request by tonyyang-svail**
Followed design at https://github.com/tonyyang-svail/pysqlflow/blob/b3034388d122bdede0b0ad51b9756eeae0a5625f/doc/client_design_doc.md

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -22,7 +22,7 @@ protoc: ## Generate python client from proto file
 	&& pip install grpcio-tools \
 	&& mkdir -p build/grpc/sqlflow/proto \
 	&& curl -o build/grpc/sqlflow/proto/sqlflow.proto \
-		https://raw.githubusercontent.com/wangkuiyi/sqlflowserver/$(SQLFLOW_VERSION)/sqlflow.proto \
```
Comment (tonyyang-svail) [inline Makefile]: wangkuiyi/sqlflow has been moved to a private repo. link to gist for now.

File: sqlflow/client.py [hunk 2]
```diff
@@ -31,21 +34,35 @@ def __init__(self, server_url=None):
...
         """
-        for res in self._stub.Run(pb.RunRequest(sql=operation)):
-            if res.WhichOneof('response') == 'messages':
-                for message in res.messages.messages:
-                    yield message
-            else:
-                yield self._decode_protobuf(res)
+        for messages in self._stub.Execute(pb.Request(sql=operation)):
+            for message in messages.messages:
+                print(message)
```
Comment (ktong) [inline sqlflow/client.py]: Please use logging instead

File: sqlflow/client.py [hunk 3]
```diff
@@ -31,21 +34,35 @@ def __init__(self, server_url=None):
...
+            query(str): SELECT .. FROM ...
+
+        Returns:
+            rowset(RowSet)
+        """
+        rowsets = []
+        for res in self._stub.Query(pb.Request(sql=query)):
+            rowsets.append(self._decode_protobuf(res))
+
+        return RowSet(rowsets=rowsets)
```
[the remark this answers is not in the document]
Comment (ktong) [reply sqlflow/client.py]: The logic in method rows should be like below:
```
def rows(query_result):
    for res in query_result:
        page = self._decode_protobuf(res)
        if not self._column_names:
          self._column_names = page.column_names
       yield from res.page.rows
```

File: sqlflow/magic.py [hunk 4]
```diff
@@ -17,15 +17,28 @@ def __init__(self, shell):
...
-    def execute(self, line, cell):
-        """Runs SQL result against a sqlflow server, specified by server_url
+    @cell_magic('sqlflow_query')
+    def query(self, line, cell):
+        """Query
 
         Example:
 
-            %%sqlflow  SELECT * FROM mytable
+            %%sqlflow_query
```
Comment (ktong) [inline sqlflow/magic.py]: %%sqlflow query

File: sqlflow/magic.py [hunk 5]
```diff
@@ -17,15 +17,28 @@ def __init__(self, shell):
...
+        """
+        assert len(line) == 0
+        return self.client.query(cell)
+
+    @cell_magic('sqlflow_execute')
+    def execute(self, line, cell):
+        """Execute
+
+        Example:
+            %%sqlflow_execute
```
Comment (ktong) [inline sqlflow/magic.py]: %%sqlflow execute

File: sqlflow/magic.py [hunk 6]
```diff
@@ -17,15 +17,28 @@ def __init__(self, shell):
...
+    def query(self, line, cell):
+        """Query
 
         Example:
 
-            %%sqlflow  SELECT * FROM mytable
+            %%sqlflow_query
+            SELECT * FROM mytable
+        """
+        assert len(line) == 0
```
Comment (ktong) [inline sqlflow/magic.py]: Why need this?

File: sqlflow/magic.py [hunk 7]
```diff
@@ -35,19 +48,8 @@ def execute(self, line, cell):
...
-                print(res)
-            else:
-                raise ValueError("can't print {}:{}".format(type(res), res))
-
-    @staticmethod
-    def print_table(table):
-        print(table["column_names"])
-        for row in table["rows"]:
-            print(row)
+        assert len(line) == 0
```
Comment (ktong) [inline sqlflow/magic.py]: Why need this?

**Pull request closed without merging**